### PR TITLE
[4.0] More namespacing

### PR DESCRIPTION
**Pull request by brianteeman**
I can only assume that these have reappeared because of merging j3

### Summary of Changes
JText => Text
JLog => Log
JHTML => HTMLHelper
JRoute => Route

### Testing Instructions
Code review  (@quy just what you like)

NOTE: I deliberately ignored anything to do with com_actionlogs, com_csp, com_privacy as they need much more namespacing and I will do each of them separately to make testing easier.  The same is true for com_finder but some might have slipped through

### CODE REVIEW

File: plugins/authentication/cookie/cookie.php [hunk 1]
```diff
@@ -54,8 +54,8 @@ public function onPrivacyCollectAdminCapabilities()
 		$this->loadLanguage();
 
 		return array(
-			JText::_('PLG_AUTHENTICATION_COOKIE') => array(
-				JText::_('PLG_AUTH_COOKIE_PRIVACY_CAPABILITY_COOKIE'),
+			Jext::_('PLG_AUTHENTICATION_COOKIE') => array(
```
Comment (Quy) [inline plugins/authentication/cookie/cookie.php]:
```suggestion
			Text::_('PLG_AUTHENTICATION_COOKIE') => array(
```

File: components/com_finder/tmpl/search/default_form.php [hunk 2]
```diff
@@ -9,18 +9,22 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Log\Log;
```
Comment (Quy) [inline components/com_finder/tmpl/search/default_form.php]:
```suggestion
use Joomla\CMS\Router\Route;
```

File: components/com_finder/tmpl/search/default_result.php [hunk 3]
```diff
@@ -9,6 +9,8 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Router\Route;
```
Comment (Quy) [inline components/com_finder/tmpl/search/default_result.php]:
```suggestion
use Joomla\CMS\Language\Text;
use Joomla\CMS\Router\Route;
```

File: components/com_finder/tmpl/search/default_results.php [hunk 4]
```diff
@@ -9,6 +9,9 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\HTML\HTMLHelper;
```
Comment (Quy) [inline components/com_finder/tmpl/search/default_results.php:12]:
```suggestion
use Joomla\CMS\HTML\HTMLHelper;
use Joomla\CMS\Language\Text;
```

File: components/com_finder/tmpl/search/default_results.php [hunk 5]
```diff
@@ -19,7 +22,7 @@
 			<?php $uri = JUri::getInstance($this->query->toUri()); ?>
 			<?php $uri->setVar('q', $this->suggested); ?>
 			<?php // Compile the suggested query link. ?>
-			<?php $linkUrl = JRoute::_($uri->toString(array('path', 'query'))); ?>
+			<?php $linkUrl = Route::_($uri->toString(array('path', 'query'))); ?>
 			<?php $link = '<a href="' . $linkUrl . '">' . $this->escape($this->suggested) . '</a>'; ?>
 			<?php echo JText::sprintf('COM_FINDER_SEARCH_SIMILAR', $link); ?>
```
Comment (Quy) [inline components/com_finder/tmpl/search/default_results.php]:
```suggestion
			<?php echo Text::sprintf('COM_FINDER_SEARCH_SIMILAR', $link); ?>
```

**Comment by brianteeman** [comment]
Thanks @quy

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**